rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -6802,115 +6804,245 @@ void getQueuedPegoutsCount_after_RSKIP271_activation() throws IOException {
         assertEquals(2, bridgeSupport.getQueuedPegoutsCount());
     }
 
-    @Test
-    void getEstimatedFeesForNextPegOutEvent_before_RSKIP271_activation() throws IOException {
-        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
-        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(false);
-
-        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+    private static Stream<Arguments> getEstimatedFeesForNextPegOutEventArgsProvider_pre_RSKIP271(BridgeRegTestConstants bridgeConstantsRegtest) {
+        List<FederationMember> members = FederationMember.getFederationMembersFromKeys(
+            PegTestUtils.createRandomBtcECKeys(7)
+        );
 
-        BridgeSupport bridgeSupport = bridgeSupportBuilder
-            .withActivations(activations)
-            .withProvider(provider)
-            .build();
+        Federation standardFed = bridgeConstantsRegtest.getGenesisFederation();
 
-        verify(provider, never()).getFeePerKb();
-        assertEquals(Coin.ZERO, bridgeSupport.getEstimatedFeesForNextPegOutEvent());
-    }
+        ActivationConfig.ForBlock preRSKIP271_activations = mock(ActivationConfig.ForBlock.class);
+        when(preRSKIP271_activations.isActive(ConsensusRule.RSKIP271)).thenReturn(false);
+        when(preRSKIP271_activations.isActive(ConsensusRule.RSKIP385)).thenReturn(false);
 
-    @Test
-    void getEstimatedFeesForNextPegOutEvent_after_RSKIP271_activation() throws IOException {
-        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
-        when(activations.isActive(ConsensusRule.RSKIP271)).thenReturn(true);
+        Federation p2shFed = new P2shErpFederation(
+            members,
+            Instant.now(),
+            1L,
+            bridgeConstantsRegtest.getBtcParams(),
+            bridgeConstantsRegtest.getErpFedPubKeysList(),
+            bridgeConstantsRegtest.getErpFedActivationDelay(),
+            preRSKIP271_activations
+        );
 
-        int pegoutRequestsCount = 5;
-        Coin feePerKB = Coin.MILLICOIN;
-        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
-        when(provider.getReleaseRequestQueueSize()).thenReturn(pegoutRequestsCount);
-        when(provider.getFeePerKb()).thenReturn(feePerKB);
+        Stream<Arguments> preRskip385 = Stream.of(
```

Comment:
pre 271 or pre 385?

Reply:
It's a mistake. It's `pre RSKIP271`. Fixed.